src/Database/Traits/Sortable.php

```diff
@@ -57,9 +57,10 @@ public function setSortableOrder($itemIds, $itemOrders = null)
             throw new Exception('Invalid setSortableOrder call - count of itemIds do not match count of itemOrders');
         }
 
+        $primaryKey = (!empty($this->primaryKey) ? $this->primaryKey : 'id');
         foreach ($itemIds as $index => $id) {
             $order = $itemOrders[$index];
-            $this->newQuery()->where('id', $id)->update([$this->getSortOrderColumn() => $order]);
+            $this->newQuery()->where($primaryKey, $id)->update([$this->getSortOrderColumn() => $order]);
```

Comment:
This line should read `$this->newQuery()->where($this->getKeyName(), $id)->update([$this->getSortOrderColumn() => $order]);`